src/classes/groundplaces.ts

```diff
@@ -354,19 +432,28 @@ export class GroundPlacesController {
         `Impossible to remove the StopGroup with the Gpuid "${stopGroupGpuidToRemove}". Make sure that the StopGroup you want to remove will not be without any StopCluster parent after this operation.`,
       );
     }
+
+    if (removeActionHistory !== RemoveActionHistory.TRUE) {
+      this.storageService.addGroundPlaceActionHistory(stopGroupGpuidToRemove, {
+        type: ActionType.REMOVE_STOP_GROUP_FROM_STOP_CLUSTER,
+        from: stopClusterGpuidParent,
+      });
+    }
   }
 
   /**
    * @description Move a StopGroup from a StopCluster to another StopCluster.
    * @param {StopGroupGpuid} stopGroupToMoveGpuid - Ground place unique identifier of the StopGroup to move.
    * @param {StopClusterGpuid} fromStopClusterGpuid - Ground place unique identifier of the old StopCluster.
    * @param {StopClusterGpuid} intoStopClusterGpuid - Ground place unique identifier of the new StopCluster.
+   * @param {RemoveActionHistory} removeActionHistory - If you don't want to have ActionHistory from this method, you can deactivate it here.
```

Comment:
i really dont understand this param and its naming. If its not true its added to the actionHistory ? when is it false. I think the previous naming was clear enough

Reply:
For this param, we need to use `RemoveActionHistory.TRUE` to not added the ActionHistory
If it's not clear, I can change it for `CreateActionHistory.FALSE`?

Reply:
i think its misleading because you're not removing anything. Yes i think `CreateActionHistory` is a lot better

Reply:
You're right, I'll change this

Reply:
Done!